css/index.css

```diff
@@ -143,4 +143,71 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
```

Comment:
Here is correct, you only want to override in the child what changed from the parent relationships once you have it setup properly in the CSS above.  Also, if you want you can remove the .middle-content .boxes .boxX, you are targeting unique classes.